docker/ci/build

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -eux
+
+BUNDLER_PATH=.bundle/gems
+
+function build_and_test_image() {
+  container_name=$1
+
+  : ===
+  : === run linter on the docker files
+  : ===
+   bundle exec puppet-docker local-lint $container_name
```

Comment:
Same here (and for any other vars where expansion to multiple args wouldn't be acceptable), i.e. `"$container_name"`